tests/ed25519-key-der.spec.js

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {util: {binary: {base58}}} = require('node-forge');
+const {_derEncode: privateDerEncode} =
+  require('../lib/ed25519PrivateKeyNode12');
+const {_derEncode: publicDerEncode} = require('../lib/ed25519PublicKeyNode12');
```

Comment:
The karma tests are failing because the *Node12 files are ignored.  It looks like `privateDerEncode` and `publicDerEncode` could be moved out of those files into a generic helper file which could be used here.  That would let this file be independent of node vs browser issues.

Reply:
OK, I'll make that mod.